src/components/dropdown/CustomRender.tsx

```diff
@@ -135,8 +134,8 @@ export const ClearIndicator = (props: any) =>
 
 export const Control = ({ children, selectProps, ...props }: any) => {
   const {iconName} = selectProps;
-  return (<div className="tk-input-group__header tk-pt-h">
-    <components.Control {...props} className="tk-input__container">
+  return (<div>
```

Comment:
Do we need to wrap it in a div?

Reply:
Yes we do, otherwise, the styles break ! I just removed the unnecessary classes here